tests/test_hvac_csc.py

```diff
@@ -0,0 +1,275 @@
+# (https://www.lsst.org).
+# See the COPYRIGHT file at the top-level directory of this distribution
+# for details of code ownership.
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program.  If not, see <https://www.gnu.org/licenses/>.
+
+import asynctest
+import logging
+
+from lsst.ts import salobj
+from lsst.ts import hvac
+
+STD_TIMEOUT = 2  # standard command timeout (sec)
+
+logging.basicConfig(format="%(asctime)s:%(levelname)s:%(name)s:%(message)s", level=logging.DEBUG)
+
+
+class CscTestCase(salobj.BaseCscTestCase, asynctest.TestCase):
+    def basic_make_csc(self, initial_state, config_dir, simulation_mode, **kwargs):
+        return hvac.HvacCsc(
+            initial_state=initial_state, config_dir=config_dir, simulation_mode=simulation_mode,
+        )
+
+    async def test_standard_state_transitions(self):
+        async with self.make_csc(initial_state=salobj.State.STANDBY, config_dir=None, simulation_mode=1):
+            await self.check_standard_state_transitions(
+                enabled_commands=(
+                    "chiller01P01",
+                    "crack01P02",
+                    "fancoil01P02",
+                    "manejadoraLower01P05",
+                    "manejadoraSblancaP04",
+                    "vea01P01",
+                    "vea01P05",
+                    "vea08P05",
+                    "vea09P05",
+                    "vea10P05",
+                    "vea11P05",
+                    "vea12P05",
+                    "vea13P05",
+                    "vea14P05",
+                    "vea15P05",
+                    "vea16P05",
+                    "vea17P05",
+                    "vec01P01",
+                    "vex03P04",
+                    "vex04P04",
+                    "vin01P01",
+                ),
+            )
+
+    async def test_bin_script(self):
+        await self.check_bin_script(name="HVAC", index=None, exe_name="run_hvac.py")
+
+    async def test_do_chiller01P01(self):
+        async with self.make_csc(initial_state=salobj.State.STANDBY, config_dir=None, simulation_mode=1):
+            await salobj.set_summary_state(remote=self.remote, state=salobj.State.ENABLED)
+            setpoint_activo = 90
+            comando_encendido = True
+            await self.remote.cmd_chiller01P01.set_start(
+                setpointActivo=setpoint_activo, comandoEncendido=comando_encendido, timeout=STD_TIMEOUT,
+            )
```

Comment:
I strongly suggest you also test that the CSC did what you asked it to (for this test and the others below). In detail I suggest that you:

1) Read the appropriate event and/or telemetry topics, e.g. with `await self.assert_next_sample(...)`. That tests that you are reporting state changes correctly.

At risk of stating the obvious, if you test events then you have to read (and, preferably, check) the initial state as well. And that is best done before you send the command. For telemetry it is probably better to flush old samples, but you will probably have to skip one and test the 2nd to avoid a race condition.

2) Check that the mock controller's state changed as expected. That checks that the CSC sent the necessary command to the mock controller.

This step is probably unnecessary if the SAL events and telemetry are output as a result of data from the mock controller (which is a nice way to operate, if the low-level controller API supports it).